src/ssl.c

```diff
@@ -42551,11 +42539,6 @@ int wolfSSL_X509_NAME_print_ex(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
         return WOLFSSL_SUCCESS;
 #endif /* WOLFSSL_APACHE_HTTPD || OPENSSL_ALL || WOLFSSL_NGINX */
     }
-    else if (flags == XN_FLAG_RFC2253) {
```

Comment:
Great catch on the double compare (EG ```if (flags == (XN_FLAG_RFC2253 & ~XN_FLAG_DN_REV))``` is the same comparison as ```if (flags == XN_FLAG_RFC2253)``` however if you notice there is unique behavior dependent upon ```#if defined(WOLFSSL_APACHE_HTTPD) || defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX)``` so should we not preserve that?

Suggested change here is to move the logic from dead code section into "else" condition inside the first if().

```
#if defined(WOLFSSL_APACHE_HTTPD) || defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX)
....
#else
        if (wolfSSL_BIO_write(bio, name->name + 1, name->sz - 2)		
                                                                 != name->sz - 2)		
             return WOLFSSL_FAILURE;
#endif /* WOLFSSL_APACHE_HTTPD || OPENSSL_ALL || WOLFSSL_NGINX */
```

Reply:
Maybe we should also simplify the compare, not sure why we have a complex comparison ```if (flags == (XN_FLAG_RFC2253 & ~XN_FLAG_DN_REV))``` when it could more simply be ```if (flags == XN_FLAG_RFC2253)```